packages/wallet/src.ts/index.ts

```diff
@@ -220,6 +289,68 @@ export class Wallet extends Signer implements ExternallyOwnedAccount, TypedDataS
 		return encryptKeystore(this, password, options, progressCallback);
 	}
 
+	async call(txRequest : Deferrable<TransactionRequest>, blockTag?: BlockTag | Promise<BlockTag>): Promise<string> {
+		this._checkProvider("call");
+		const tx = await resolveProperties(this.checkTransaction(txRequest));
+		const contractAccountLikeID = getAccountFromAddress(tx.to.toString());
+		const contractId = `${contractAccountLikeID.shard}.${contractAccountLikeID.realm}.${contractAccountLikeID.num}`;
+
+		const thisAcc = getAccountFromAddress(await this.getAddress());
+		const thisAccId = `${thisAcc.shard}.${thisAcc.realm}.${thisAcc.num}`;
```

Comment:
Lets implement new method in `address/index.ts` : `asAccountString(AccountLike)` which will expose the logic